lib/Conversion/Utils/Utils.cpp

```diff
@@ -237,6 +237,19 @@ SmallVector<Value> getTypeConvertedValues(OpBuilder &b, Location loc,
   }));
 }
 
+llvm::SmallVector<int64_t> ConvertShapeToMlir(llvm::ArrayRef<int64_t> shapes) {
+  return llvm::to_vector(llvm::map_range(shapes, [](int64_t shape) {
+    return shape == kDynamicSize ? mlir::ShapedType::kDynamic : shape;
```

Comment:
Did you try using the `makeShapeLLVMCompatible` utility?

Reply:
I this copy from upstream tensorflow tosa implementation, It looks like something similar to makeShapeLLVMCompatible()

Reply:
Done